releasenotes/notes/apm-span-links-support-78478b81bf08e26f.yaml

```
@@ -0,0 +1,6 @@
+---
+enhancements:
+  - |
+    Adds support for OpenTelemetry span links to the Trace Agent otlp endpoint when converting OTLP spans (span links are added as metadata to the converted span).
```

Recommend uniform formatting.
```suggestion
    Adds support for OpenTelemetry span links to the Trace Agent OTLP endpoint when converting OTLP spans (span links are added as metadata to the converted span).
```